Mathlib/Data/Bitvec/Basic.lean

```diff
@@ -92,34 +90,41 @@ theorem ofNat_toNat {n : ℕ} (v : Bitvec n) : Bitvec.ofNat _ v.toNat = v := by
     subst n
     simp only [Bitvec.ofNat, Vector.toList_cons, Vector.toList_nil, List.reverse_cons,
       Vector.toList_append, List.foldr]
-    erw [add_lsb_div_two, to_bool_add_lsb_mod_two]
+    erw [addLsb_div_two, decide_addLsb_mod_two]
     congr
-    apply ys_ih
+    rename_i tail_ih
+    apply tail_ih
     rfl
 #align bitvec.of_nat_to_nat Bitvec.ofNat_toNat
 
+set_option pp.coercions false
 theorem toFin_val {n : ℕ} (v : Bitvec n) : (toFin v : ℕ) = v.toNat := by
-  rw [to_fin, Fin.coe_of_nat_eq_mod, Nat.mod_eq_of_lt] <;> apply to_nat_lt
+  rw [toFin]
+  rw [Fin.coe_ofNat_eq_mod']
```

Comment:
I'm pretty sure this lemma doesn't exist any more; can you rebase on master?

Reply:
In that case, deleting it is easier, since the coercion is giving me a headache. `Fin.coe_of_nat_eq_mod` doesn't exist but `Fin.coe_of_nat_eq_mod'` does

Reply:
I am not asking you to delete `toFin_val`, my comment was about `Fin.coe_of_nat_eq_mod`

Reply:
It exists on the latest  mathlib3port if that is what you mean: https://github.com/leanprover-community/mathlib3port/blob/1107b9799e5841267f0cc8d3ef34f79c0180fa45/Mathbin/Data/Bitvec/Basic.lean#L103

Reply:
> but `Fin.coe_of_nat_eq_mod'` does

Not in master it doesn't.

Reply:
Oh. I see what you mean. Will rebase right away

Reply:
Done. a dsimp fixed the coercion

Reply:
It looksk like you made the mistake of rebasing then merging, which makes a mess of the history. After rebasing, you should `git push --force-with-lease`

Reply:
I recommend you
* `git reset --hard 31c562ccfed156aaab8741b2434f6833ad21ac9e`
* `git push --force-with-lease`

Reply:
You can then `git cherry-pick 93175c3a82ec5c03df1115f7a09bafea0173ffca` to recover the commit you just made after I wrote that comment.

Reply:
Okay, but git rebase + merge seems to have put my commits after the commits on the master branch. Is this incorrect?

Reply:
@eric-wieser I have a successful build. The history seems fine now. All the changes I did come after the changes to master on which I rebased. Is this correct?

Reply:
I did this, but I don't see this push

Reply:
Looks good now, thanks